int32 id

# Contains everything needed to reconstruct a Sample object
DataType[] sensor_data
int8 arm_datatype # which arm is used for trials 
